core/src/main/java/nl/nn/adapterframework/lifecycle/IbisApplicationInitializer.java

```diff
@@ -114,21 +113,6 @@ public WebApplicationContext initWebApplicationContext(ServletContext servletCon
 		}
 	}
 
-	//TODO: remove this in 8.0
```

Comment:
Perhaps a bit tricky to remove this in the PR for removing BTM although I agree that this code should be removed.
Perhaps better to do that in another PR, there's a lot more WAS related code that needs to be removed.

Reply:
I agree. Done! #5525